examples/http/tdag_pairs.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [[ -z "$1" ]]; then
+	echo "USAGE: ./tdag_pairs.sh /path/to/dir/with/raw_http_test_cases"
+	exit 1
+fi
+
+if [[ ! -d "$1" ]]; then
+	echo "ERROR: directory does not exist"
+	exit 2
+fi
+
+mkdir -p results
+for file in "$1"/*; do
+	TEST_CASE=$(basename "$file")
+	if [[ ! -d results/"$TEST_CASE" ]]; then
```

Comment:
When I was working on the httpd harness initially, there were some test cases that didn't process correctly, and I added this to avoid having to reprocess everything. I retained this functionality here because it would be useful if one had already done an evaluation over an existing corpus, then expanded the corpus and wanted to evaluate new test cases under the existing set of parsers. However, if new parsers were added on line 25 below, unless the test case subdirectory is deleted, the new parsers would not be evaluated over that test case.